app/workers/test_job.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class TestJob
```

Comment:
I don't see this used in the specs — was this part of your local testing? Should it be removed?

Reply:
This was added accidentally lol

Reply:
cool that's what I figured haha